src/helpers/filterActivities.js

```diff
@@ -0,0 +1,12 @@
+import capitalizeString from './stringFormatter';
+
+const filterActivities = (status, context) => {
+  const filteredActivities = context[Object.keys(context)[0]]
```

Comment:
is `context[Object.keys(context)[0]]` === context.activities ?

Reply:
They will definitely fail 😢

Reply:
Fixed, Please review @mkiterian